web/client/utils/ogc/WFST/__tests__/WFST-test.js

```diff
@@ -18,12 +17,24 @@ const expectedInsertWyoming = require('raw-loader!../../../../test-resources/wfs
 const expectedInsertmuseam = require('raw-loader!../../../../test-resources/wfst/insert/museam_1_1_0.xml');
 const expectedDelete = require('raw-loader!../../../../test-resources/wfst/delete/museam_1_1_0.xml');
 const expectedUpdate = require('raw-loader!../../../../test-resources/wfst/update/museam_1_1_0.xml');
+const doubleMuseamInsert = require('raw-loader!../../../../test-resources/wfst/insert/double_museam_1_1_0.xml');
 describe('Test WFS-T request bodies generation', () => {
     it('WFS-T insert', () => {
         const {insert} = requestBuilder(describeStates);
-        const result = insert(wyoming, describeStates);
+        const result = insert(wyoming);
         expect(result).toExist();
     });
+    it('WFS-T insert (array)', () => {
+        const {insert} = requestBuilder(describeStates);
+        const result = insert([wyoming]);
+        expect(result).toExist();
+    });
+    it('WFS-T transaction insert (arg list)', () => {
+        const {transaction, insert} = requestBuilder(describePois);
+        const result = transaction(insert(museam, museam));
+        expect(result + '\n').toBe(doubleMuseamInsert);
```

Comment:
expect(result).toBe(doubleMuseamInsert.replace(/[\r\n]/g,''))

to let windows run the tests and NOT fail

Please fix all the new tests